app.py

```diff
@@ -178,29 +192,10 @@ def charge():
     customer_first = request.form['first_name']
     customer_last = request.form['last_name']
 
-    email_is_valid = validate_email(customer_email)
-
```

Comment:
Could it be an issue if we're no longer checking for a valid email server-side? Especially since this is now an API endpoint I'd hesitate to rely on the client checking. But maybe I'm missing something.

Reply:
@x110dc I brought this back and updated the manual-testing steps above.